### DISCUSSION TITLE
how can i get relevant literature identified by Asreview?

### DISCUSSION THREAD

**Discussion by zaq0718** [asreview/asreview#1843] [Q&A] [answered]
I imported 1000 records with titles and abstracts as an Excel file into the Asreview. Below is the details of settings in my project.

![image](https://github.com/user-attachments/assets/31ce79fc-d232-4bc9-a89d-a29a7ba77b16)

I screened 11 studies. Seven of them are relevant (see the below screenshot). 
![image](https://github.com/user-attachments/assets/08fe5358-89b3-4b70-914b-19eff20e3aad)

Then, I decided to stop screening and exported the results. In the Asreview, I clicked export-->select file--> dataset relevant only. When I downloaded the dataset, it only showed these 7 relevant studies that I identified during the screening process (see the below screenshot)

![image](https://github.com/user-attachments/assets/705f2c2e-c4b9-40b8-8464-828b30026846)

When I exported the results, i also noticed that CDM showed this message:PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'C:\\Users\\YI-JHE~1\\AppData\\Local\\Temp\\tmp2mtod9_j\\export_dataset.xlsx'

I am not sure if this message is related to the exported dataset.

I would like to ask:  how can I get the rest of the relevant studies that I did not screen but  Asreview identified as relevant studies?

**Comment by Rensvandeschoot** [comment, marked as answer]
Concerning your last question, you can see the seven articles you labelled as relevant in your exported file. To get access to the seven full-text files, you can use their DOIs to find these via your institutional access platform (or use reference management systems to do this for you). If you expect more relevant papers in your dataset, you should continue labeling until you have confidence you can stop; see also this paper on a [stopping heuristic](https://doi.org/10.1186/s13643-024-02502-7) and [this discussion thread](https://github.com/asreview/asreview/discussions/557) for even more suggestions. 


Concerning your error message: ping @J535D165

**Reply by J535D165** [reply]
The export seems to have been successful in the end, so you can ignore the message in the CMD window.

**Reply by zaq0718** [reply]
Thank you both! This information is really helpful!
